[DOCS]: Getting Started changes (#114)
elastic/security-docs: docs/getting-started/security-ui.asciidoc

* Overview
* Detections
* Hosts
* Network
* Timelines
* Cases
* Administration

[float]
[[overview-ui]]
=== Overview page

The Overview page provides a high-level snapshot view of detections, external alerts, and event trends. These trends are useful to assess overall system health and find anomalies that may require further investigation.

image::images/overview-pg.png[]

*Histograms*

Time-based histograms show you the number of detections, alerts, and events that have occurred within the selected time range. You can select areas of interest in time-based histograms by selecting a region to reflect a date range, or by selecting a preset value in the timepicker. In the **Stack by** dropdown, you can select specific parameters for which to visualize the individual counts. For example, in the Detection alert trend histogram, stack by `signal.rule.name` to display the total counts by alert name within the specified time frame.

TIP: All Elastic Security histograms, graphs, and tables contain an **Inspect** button so you can examine the {es} queries used to retrieve data throughout
the app.

> Select the collapsable *Timeline* button on the rightmost side of the {security-app} to start an investigation. See <<timeline-ui-overview,>> for more information.

Reviewer: ```suggestion
Select the collapsable *Timeline* button on the rightmost side of the {security-app} to start an investigation. See <<timelines-ui>> for more information.
```

*Host and network events*

View event and host counts specific to Elastic data shippers and apps, such as **Auditbeats** or **Elastic Endpoint Security**. Expand each category to view specific counts of hosts or network events related to the selected category.

[role="screenshot"]
image::images/events-count.png[]

[float]
[[hosts-ui]]
=== Hosts page

> The Hosts view provides key metrics regarding host-related security events, and a set of data tables that let you interact with the Timeline. See <<hosts-overview,Hosts page overview>> for more information.

Reviewer: This hosts-overview page doesn't seem to exist in the book yet, so I omitted it for now:
```suggestion
The Hosts view provides key metrics regarding host-related security events, and a set of data tables that let you interact with the Timeline.
//See <<hosts-overview,Hosts page overview>> for more information.
```

Reviewer: I think this change (omission of the hosts-overview link) is still necessary.

[role="screenshot"]
image::images/hosts-pg.png[]

[float]
[[network-ui]]
=== Network page

The Network page provides key network activity metrics via an interactive map and network event tables
that enable interaction with the Timeline. See <<network-page-overview>> for more information. 

[float]
[[detection-engine-ui]]
=== Detections page

> The Detections page allows you to view and manage all alerts and detection rules to monitor activity within your network. See <<detection-engine-intro>> for more information about detections, prebuilt rules, and alerts.

Reviewer: ```suggestion
The Detections page allows you to view and manage all alerts and detection rules to monitor activity within your network. See <<detection-engine-overview>> for more information about detections, prebuilt rules, and alerts.
```

[role="screenshot"]
image::images/detections-ui.png[]

[float]
[[timelines-page]]
=== Timelines page

Use the Timelines page to investigate alerts and complex threats, such as lateral movement of malware across hosts

> in your network. Timelines are responsive and allow you to share your findings among other team members. See <<timeline-ui-overview>> for information about getting started with Timelines.

Reviewer: ```suggestion
in your network. Timelines are responsive and allow you to share your findings among other team members. See <<timelines-ui>> for information about getting started with Timelines.
```

TIP: All Elastic Security histograms, graphs, and tables contain an **Inspect** button so you can examine the {es} queries used to retrieve data throughout
the app.

Select the collapsable *Timeline* button on the rightmost side of the {security-app} to start an investigation. See <<timelines-ui,>> for more information.

*Host and network events*

View event and host counts specific to Elastic data shippers and apps, such as **Auditbeats** or **Elastic Endpoint Security**. Expand each category to view specific counts of hosts or network events related to the selected category.

[role="screenshot"]
image::images/events-count.png[]

[float]
[[hosts-ui]]
=== Hosts page

The Hosts view provides key metrics regarding host-related security events, and a set of data tables that let you interact with the Timeline. See <<hosts-overview>> for more information.

[role="screenshot"]
image::images/hosts-pg.png[]

[float]
[[network-ui]]
=== Network page

The Network page provides key network activity metrics via an interactive map and network event tables

> that enable interaction with the Timeline. See <<network-page-overview>> for more information.

Reviewer: ```suggestion
that enable interaction with the Timeline. See <<network-pg-ov>> for more information.
```

[role="screenshot"]
image::images/network-ui.png[]

[float]
[[detection-engine-ui]]
=== Detections page

The Detections page allows you to view and manage all alerts and detection rules to monitor activity within your network. See <<ddetection-engine-overview>> for more information about detections, prebuilt rules, and alerts.

[role="screenshot"]
image::images/detections-ui.png[]

[float]
[[timelines-page]]
=== Timelines page

Use the Timelines page to investigate alerts and complex threats, such as lateral movement of malware across hosts
in your network. Timelines are responsive and allow you to share your findings among other team members. See <<timelines-ui>> for information about getting started with Timelines.

[role="screenshot"]
image::images/timeline-ui.png[]

[float]
[[cases-ui]]
=== Cases page

The Cases page is used to open and track security issues directly in the {security-app}. For more information, see <<cases-overview>>.

[role="screenshot"]
image::images/cases-ui-home.png[]

[float]
[[admin-ui]]
=== Administration page

> The Administration page allows you to view and manage hosts that are running Elastic Endpoint Security. You can also manage agent integration and monitor the configuration status of your hosts to ensure they are protected. See <<admin-pg-ov>> for more information.

Reviewer: This linked page also fails for me, so I recommend omitting it until management/admin/admin-pg-ov.asciidoc is successfully included in the book:

```suggestion
The Administration page allows you to view and manage hosts that are running Elastic Endpoint Security. You can also manage agent integration and monitor the configuration status of your hosts to ensure they are protected.
// See <<admin-pg-ov>> for more information.
```

[[siem-ui-overview]]
[role="xpack"]
= Elastic Security UI overview

The {security-app} is a highly interactive workspace designed for security analysts. It provides a clear overview of events and alerts from your environment, and you can use the interactive UI to drill down into areas of interest.

[discrete]
[[search-overview]]
== Search

Filter for alerts, events, processes, and other important security data with the benefit of *{kibana-ref}/kuery-query.html[{kib} Query Language (KQL)]* in the  Search bar, which appears at the top of each page throughout the app. A date/time filter set to `Last 24 hours` is enabled by default, but can be changed to any time range. If you want to filter your search results with other fields, select **Add Filter,** followed by the field from which to filter and the operator (such `is not` or `is between`) for your query.

To save specific filters and queries, click the *Save* button, then **Save current query.**

[role="screenshot"]
image::images/siem-click-swipe.png[]

[discrete]
[[page-tabs]]
== Page tabs

> The {security-app} contains seven page tabs that enable analysts to view, analyze, and manage security data:

Reviewer 2: ```suggestion
The {security-app} contains seven page tabs that enable analysts to view, analyze, and manage security data:

```
Add a new line so the bullets render correctly.

the app.

Select the collapsable *Timeline* button on the rightmost side of the {security-app} to start an investigation. See <<timelines-ui,>> for more information.

*Host and network events*

View event and host counts specific to Elastic data shippers and apps, such as **Auditbeats** or **Elastic Endpoint Security**. Expand each category to view specific counts of hosts or network events related to the selected category.

[role="screenshot"]
image::images/events-count.png[]

[float]
[[hosts-ui]]
=== Hosts page

The Hosts view provides key metrics regarding host-related security events, and a set of data tables that let you interact with the Timeline.
// See <<hosts-overview>> for more information.

[role="screenshot"]
image::images/hosts-pg.png[]

[float]
[[network-ui]]
=== Network page

The Network page provides key network activity metrics via an interactive map and network event tables
that enable interaction with the Timeline. See <<network-page-overview>> for more information.

[role="screenshot"]
image::images/network-ui.png[]

[float]
[[detection-engine-ui]]
=== Detections page

> The Detections page allows you to view and manage all alerts and detection rules to monitor activity within your network. See <<ddetection-engine-overview>> for more information about detections, prebuilt rules, and alerts.

Reviewer: Small typo in link:
```suggestion
The Detections page allows you to view and manage all alerts and detection rules to monitor activity within your network. See <<detection-engine-overview>> for more information about detections, prebuilt rules, and alerts.
```
